show/main.py

```diff
@@ -2806,5 +2806,26 @@ def features():
         body.append([key, status_data[key]['status']])
     click.echo(tabulate(body, header))
 
+# show whether the auto-restart feature for containers or a specific container
+# is enabled or disabled.
+
+@cli.command('autorestart')
```

Comment:
Since we will have multiple container features including autorestart, this command should be nested in a group (or a couple groups), maybe `show container feature autorestart`. This way we can add other things under both the "container" and "feature" groups in the future.

Reply:
Please help me review it.